code-challenges.test.js

```diff
@@ -1,52 +1,165 @@
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
 
-// b) Create the function that makes the test pass.
+// // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
-// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
+// // a) Create a test with expect statements for each of the variables provided.
 
-// a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
 
-// b) Create the function that makes the test pass.
+describe("object1", () => {
+    it("decides if the number inside it is evenly divisible by three or not", () => {
+      expect(object1()).toEqual("15 is divisible by three")
+    })
+  })
 
-// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+describe("object1", () => {
+    it("decides if the number inside it is evenly divisible by three or not", () => {
+      expect(object2()).toEqual("0 is divisible by three")
+    })
+  })
 
-// a) Create a test with expect statements for each of the variables provided.
+describe("object3", () => {
+    it("decides if the number inside it is evenly divisible by three or not", () => {
+      expect(object1()).toEqual("-7 is not divisible by three")
+    })
+  })
+
+// //   // Received a good error
+// // //   object1 › decides if the number inside it is evenly divisible by three or not
+
+// // //     TypeError: object1 is not a function
+
+// // //       26 | describe("object1", () => {
+// // //       27 |     it("decides if the number inside it is evenly divisible by three or not", () => {
+// // //     > 28 |       expect(object1()).toEqual("15 is divisible by three")
+// // //          |              ^
+
+
+
+// // // b) Create the function that makes the test pass.
+
+// // // Psuedocode
+// // // Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
+// // // Input: object1, object2, object3
+// // // Output: If object1 is divisible by three - return "object 1 is divisible by three
+// // //         If object1 is not divisible by three - return "object 1 is not divisible by three"
+// // //         Need an if statemeny to determine whether each object is divisible by three  
+// // //         Use string interpolation for object1, object2, and object3
+
+    
+const object1 = { number: 15 }
+const object2 = { number: 0 }
+const object3 = { number: -7 }
+divideBy3 = () => {
```

Comment:
By setting this function up with a parameter that takes in any object when invoked, you can make this much more dynamic.  Once you have the object available as a parameter, you can use dot notation to access the value (ex: object.number)